tests/Load/utils/scenarioUtils.js

```diff
@@ -0,0 +1,100 @@
+import {ScenariosBuilder} from './scenariosBuilder.js'
+import {ThresholdsBuilder} from "./thesholdsBuilder.js";
+
+export class ScenarioUtils {
+    constructor(utils, scenarioName) {
+        this.utils = utils;
+        this.config = utils.getConfig();
+        this.smokeConfig = this.config.endpoints[scenarioName].smoke;
+        this.averageConfig = this.config.endpoints[scenarioName].average;
+        this.stressConfig = this.config.endpoints[scenarioName].stress;
+        this.spikeConfig = this.config.endpoints[scenarioName].spike;
+        this.setupTimeout =
+            this.config.endpoints[scenarioName].setupTimeoutInMinutes + 'm';
+        this.delay = this.config.delayBetweenScenarios;
+        this.averageTestStartTime = 0;
+        this.stressTestStartTime = 0;
+        this.spikeTestStartTime = 0;
+    }
+
+    getOptions() {
+        return {
+            setupTimeout: this.setupTimeout,
+            insecureSkipTLSVerify: true,
+            scenarios: this.getScenarios(),
+            thresholds: this.getThresholds()
+        }
+    }
+
+    getScenarios() {
```

Comment:
this functions looks like it duplicate, need to refactor this function as explained above